sites/public/styles/overrides.scss

```diff
@@ -111,29 +155,124 @@
   .alert-box.primary {
     background-color: var(--bloom-color-gray-400);
   }
-}
 
-:root {
-  --detroit-font-sans: [ "Montserrat", "Open Sans", "Helvetica", "Arial", "Verdana", "sans-serif" ];
-  --detroit-font-serif: [ "Montserrat", "Droid Serif", "Georgia", "Times", "serif" ];
+  .filter-form {
+    .view-item__label {
+      --label-text-color: var(--bloom-color-gray-750);
+    }
+  }
 
-  --bloom-font-sans: var(--detroit-font-sans);
-  --bloom-font-alt-sans: var(--detroit-font-sans);
-  --bloom-color-accent-cool: #297e73;
-  --bloom-color-gray-700: #000000;
-  --bloom-color-gray-800: #18252A;
-  --bloom-color-gray-950: #000000;
-  --bloom-color-lush: #feb70d;
-  --bloom-color-primary-dark: rgb(0, 68, 69);
-  --bloom-color-primary-darker: #004445;
-  --bloom-color-primary-light: #F2F2F2;
-  --bloom-color-primary-lighter: #F2F2F2;
-  --bloom-color-primary: rgb(41, 126, 115);
-  --bloom-color-warn: #feb70d;
-  --bloom-font-serif: var(--detroit-font-serif);
-  --primary-appearance-hover-background-color: white;
-  --primary-appearance-hover-label-color: var(--bloom-color-primary);
-  --primary-appearance-hover-border-color: var(--bloom-color-primary);
-  --outlined-appearance-hover-background-color: var(--bloom-color-primary);
-  --outlined-appearance-hover-border-color: var(--bloom-color-primary);
+  .drawer {
+    --content-padding: 0;
+    --content-height: 100%;
+  }
+
+  .page-header {
+    --inverse-background-color: var(--bloom-color-primary-darker);
+    --title-font-size-desktop: 1.75rem;
+    --title-font-size-mobile: 1.5rem;
+    h1 {
+      font-weight: 700;
+    }
+  }
+
+  .footer-sock {
+    --background-color: var(--bloom-color-black);
+    --copyright-text-color: var(--bloom-color-gray-600);
+  }
+
+  .button-group {
+    --background-color: var(--bloom-color-gray-300);
+  }
+
+  .details {
+    --desktop-width: auto;
+  }
+
+  .markdown {
+    h3 {
+      margin-bottom: var(--bloom-s2);
+      margin-top: var(--bloom-s8);
+      font-size: var(--bloom-font-size-xl);
+    }
+    h4 {
+      font-size: var(--bloom-font-size-lg);
+      margin-bottom: var(--bloom-s2);
+      margin-top: var(--bloom-s8);
+    }
+  }
+
+  .icon-black {
+    &:hover {
+      svg {
+        fill: var(--bloom-color-black);
+      }
+    }
+    svg {
+      fill: var(--bloom-color-black);
+    }
+  }
+
+  .drawer__header {
+    h1 {
+      font-size: 1.375rem;
+    }
+  }
+
+  .field {
+    .prepend {
+      font-size: var(--bloom-font-size-xl);
+    }
+  }
+
+  .info-card {
+    font-size: var(--bloom-font-size-sm);
+  }
+
+  .favorites-layout {
+    .loading-overlay {
+      --padding: var(--bloom-s8) 0 var(--bloom-s14) 0;
+    }
+  }
+
+  .home-page-hero {
+    --inner-background-color: rgba(255, 255, 255, 0.9);
```

Comment:
Definitely not something we need to do as part of this. Wondering if this is a use case we would want to make available as an option in ui-c. If the hero component is not going to be part of seeds I don't think it's worth investigating though

Reply:
The inner-background-color is a uic variable already - since we have multiple instances of the hero component and only one of them wanted this override, I had to add the custom home-page-hero flavor for this override.